cmake/modules/StandaloneOverlay.cmake

```diff
@@ -22,16 +22,13 @@ set(SWIFT_DARWIN_DEPLOYMENT_VERSION_WATCHOS "2.0" CACHE STRING "")
 set(SWIFT_ENABLE_PARSEABLE_MODULE_INTERFACES TRUE)
 
 set(SWIFT_SOURCE_DIR "${SWIFT_SOURCE_ROOT}/src/swift" CACHE PATH "")
-set(SWIFT_PATH_TO_LLVM_SOURCE "${SWIFT_SOURCE_ROOT}/src/llvm" CACHE PATH "")
 set(SWIFT_NATIVE_SWIFT_TOOLS_PATH "${TOOLCHAIN_DIR}/usr/bin" CACHE PATH "")
 set(SWIFT_SDKS ${SWIFT_HOST_VARIANT_SDK})
 
-set(LLVM_CMAKE_MODULES_PATH "${SWIFT_PATH_TO_LLVM_SOURCE}/cmake/modules")
+find_package(LLVM CONFIG REQUIRED NO_DEFAULT_PATH NO_CMAKE_FIND_ROOT_PATH)
+list(APPEND CMAKE_MODULE_PATH "${LLVM_CMAKE_DIR}")
```

Comment:
Why not combine these two append operations into one?

Reply:
hmm, how would you do that?  One is a `find_package` and the other is a list modification.

Reply:
I mean the 2 consecutive `CMAKE_MODULE_PATH` append operations.

Reply:
Oh!  I see what you mean.  Mostly because the second one made my brain hurt, no real reason.  I can adjust that to your liking :-).